scripts/set_examples_to_doc.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Add docstring, please. It should explain what the script for and how to run this.